consensus-types/blocks/setters.go

```diff
@@ -172,3 +173,12 @@ func (b *SignedBeaconBlock) SetBlobKzgCommitments(c [][]byte) error {
 	b.block.body.blobKzgCommitments = c
 	return nil
 }
+
+// SetExecutionRequests sets the execution requests in the block.
+func (b *SignedBeaconBlock) SetExecutionRequests(er *enginev1.ExecutionRequests) error {
```

Comment:
nitpick: `er` is an awkward var name (so close to `err`), seems like `req` is the default elsewhere.